### PR TITLE
Fix/uppsf 666 update docker image

### PR DESCRIPTION
**Pull request by atanaskuzmanov-ft**
Updating the docker image due to GO Security Vulnerability:
- For the Go-based services, we are migrating all of our Docker files to multi-stage builds (if not in place already) using `golang:1` as a base image and `scratch` for the runtime.
- The same `golang:1` is used for CircleCI image.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: /go/src/github.com/Financial-Times/concept-exporter
     docker:
-      - image: golang:1.8.3
+      - image: golang:1
         environment:
           GOPATH: /go
           CIRCLE_TEST_REPORTS: /tmp/test-results
```
Comment (mchompalova) [inline .circleci/config.yml]: Please update the Docker version as well.

File: Dockerfile [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
 
+# Multi-stage build - copy certs and the binary into the image
+FROM scratch
 WORKDIR /
+COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=0 /artifacts/* /
 
-CMD [ "/concept-exporter" ]
\ No newline at end of file
+CMD [ "/concept-exporter" ]
```
Comment (mchompalova) [inline Dockerfile]: Please add a new line.

File: Dockerfile [hunk 3]
```diff
@@ -1,31 +1,32 @@
...
-  && BUILDINFO_PACKAGE="${ORG_PATH}/${PROJECT}/vendor/${ORG_PATH}/service-status-go/buildinfo." \
-  && VERSION="version=$(git describe --tag --always 2> /dev/null)" \
+ENV ORG_PATH="github.com/Financial-Times"
+ENV SRC_FOLDER="${GOPATH}/src/${ORG_PATH}/${PROJECT}"
+ENV BUILDINFO_PACKAGE="${ORG_PATH}/${PROJECT}/vendor/${ORG_PATH}/service-status-go/buildinfo."
+
+COPY . ${SRC_FOLDER}
+WORKDIR ${SRC_FOLDER}
+
+# Install dependancies
```
Comment (mchompalova) [inline Dockerfile]: dependencies :) 
Sorry to be the grammar police :)

File: Dockerfile (hunk 2 above)
Comment (atanaskuzmanov-ft) [reply Dockerfile]: New line was there on my local IDE, no idea why it was not in the commit.
Fixed now.

File: Dockerfile (hunk 3 above)
Comment (atanaskuzmanov-ft) [reply Dockerfile]: No worries, didn't notice that it had a typo on this one.
Fixed.

File: .circleci/config.yml (hunk 1 above)
Comment (atanaskuzmanov-ft) [reply .circleci/config.yml]: Fixed.

**Pull request merged**